src/randoop/reflection/ReflectionPredicate.java

```diff
@@ -11,10 +11,36 @@
  */
 public interface ReflectionPredicate {
 
+  /**
+   * Indicate whether a class meets the use criteria for predicate.
+   * 
+   * @param c  the class to test.
+   * @return true if the class meets the predicate criteria, false, otherwise.
+   */
   public boolean canUse(Class<?> c);
```

Comment:
Calls to the canUse methods are often wrapped in a call named "satisfies".  Would it be better to use consistent naming and rename these methods as well?